### DISCUSSION TITLE
"--help" output for "Plugin options:" section seems to be a bit disordered in version 6.6.0

### DISCUSSION THREAD

**Discussion by thisisgk** [streamlink/streamlink#5837] [Q&A]
### Description

I post it here because it might not be an issue to fix.

Compared to version 6.5.1, `--help` output for `Plugin options:` section seems to be a bit disordered in 6.6.0.

1. The order of options for each plugin is shown in reverse. 
For ex. Afreeca plugin, in 6.5.1,
`--afreeca-username`, `--afreeca-password`, then `--afreeca-purge-credentials`,
in 6.6.0,
`--afreeca-purge-credentials`, `--afreeca-password`, then `--afreeca-username`.
2. `==SUPPRESS==` is newly shown in 6.6.0 for some plugins. For ex. Sbscokr plugin.

**Comment by bastimeyer** [comment]
> I post it here because it might not be an issue to fix.

No, this is a bug. Please open an issue on the issue tracker. Going to close this here, because the discussion forum is not meant for bug reports.
